src/VolleyManagement.Backend/VolleyManagement.Services/GameService.cs

```diff
@@ -317,7 +317,8 @@ public void SwapRounds(int tournamentId, byte firstRoundNumber, byte secondRound
         /// <param name="tournamentId">Identifier of the tournament</param>
         public void RemoveAllGamesInTournament(int tournamentId)
         {
-            var gamesToRemove = GetTournamentResults(tournamentId);
+           // var gamesToRemove = GetTournamentResults(tournamentId);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS125)